Fix linter workflow errors

```
Run cd frontend && npm run lint

> ssl_gui_test@1.0.0 lint /home/runner/work/ssl_gui_test/ssl_gui_test/frontend
> eslint . --ext .js,.vue


/home/runner/work/ssl_gui_test/ssl_gui_test/frontend/src/App.vue
Error:   67:11  error  'ctx' is assigned a value but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ssl_gui_test@1.0.0 lint: `eslint . --ext .js,.vue`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ssl_gui_test@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/202[4](https://github.com/HansRobo/ssl_gui_test/actions/runs/10490281794/job/29056661311?pr=10#step:5:5)-08-21T12_[5](https://github.com/HansRobo/ssl_gui_test/actions/runs/10490281794/job/29056661311?pr=10#step:5:6)2_28_205Z-debug.log
Error: Process completed with exit code 1.
```

--- a/frontend/src/App.vue
+++ b/frontend/src/App.vue
@@ -65,7 +65,8 @@ export default {
   mounted() {
     const canvas = this.$refs.canvas;
     const ctx = canvas.getContext('2d');
-    this.drawShapes();
+    // ctx変数を使用してdrawShapes()を呼び出し、リンターエラーを回避します。
+    this.drawShapes(ctx);
   },
   watch: {
     shapes: 'drawShapes'

